### PR TITLE
fix(charts): big-number display broken in echarts

### PR DESCRIPTION
**Pull request by anamitraadhikari**
### SUMMARY
The current implementation of the time-series charts in ECharts seems to have a limitation in properly rendering big numbers. As a result, the display of big numbers in the chart is broken.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

Before:
68d58a51-b0f2-42b0-b191-f35372d8b5dd

After:
19053635-9d5a-42e4-81a0-cd2d7aa51790

### TESTING INSTRUCTIONS
To replicate the issue with big number display in time series charts, you can create a time series chart that includes data points with large numeric values. Here is an example SQL query that can be executed in SQL Lab to generate such data:

```
select 9223372036854775807 as huge_number, 'abc' as str union all
select 9223372036854775807.5 as huge_number, 'foo' as str union all
select 92233720368547758071111 as huge_number, 'bar' as str
```

1. After executing the SQL query in SQL Lab and getting the result set, click on the "Create Chart" button.
2. In the chart creation page, switch to the "LINE CHART" option.
3. On the chart configuration panel, select the 'str' column for the x-axis.
4. Add the "SUM(huge_number)" column to the y-axis to represent the aggregated big numbers.
5. Update the chart to apply the changes and visualize the data.
6. By performing these steps, you will be able to replicate the issue where the ECharts library struggles to properly render the big number values on the time series chart.

### CODE REVIEW

File: superset-frontend/plugins/plugin-chart-echarts/src/types.ts [hunk 1]
```diff
@@ -168,4 +171,21 @@ export interface TreePathInfo {
...
+  constructor(props: any) {
+    const { metadata } = props;
+    const mergedProps = {
+      ...props,
+      metadata: new ChartMetadata({
+        parseMethod: 'json',
+        ...metadata,
+      }),
+    };
+    super(mergedProps);
```
Comment (villebro) [inline superset-frontend/plugins/plugin-chart-echarts/src/types.ts:187]: nit: Maybe we don't need to create the intermediate `mergedProps`, but could just inline it in `super()`

File: superset-frontend/plugins/plugin-chart-echarts/test/Timeseries/index.test.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+import { EchartsTimeseriesChartPlugin } from '@superset-ui/plugin-chart-echarts';
+
+describe('@superset-ui/plugin-chart-echarts', () => {
+  it('exists', () => {
+    const chart = new EchartsTimeseriesChartPlugin().configure({
+      key: 'echarts_timeseries',
+    });
+    expect(chart.metadata.parseMethod).toEqual('json');
+  });
+});
```
Comment (villebro) [inline superset-frontend/plugins/plugin-chart-echarts/test/Timeseries/index.test.ts:28]: Looking at the test suite, maybe we could simplify and make this more comprehensive by moving this test into `test/index.test.ts` and have a section there that ensures `parseMethod` is correctly defined for all plugins.

Bycatch: I also noticed that the `exists` test there doesn't cover all plugins. While you're at it, can you add the missing plugins, too?

File: superset-frontend/plugins/plugin-chart-echarts/test/Timeseries/index.test.ts [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
++ * software distributed under the License is distributed on an
++ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
++ * KIND, either express or implied.  See the License for the
++ * specific language governing permissions and limitations
++ * under the License.
++ */
+import { EchartsTimeseriesChartPlugin } from '@superset-ui/plugin-chart-echarts';
+
+describe('@superset-ui/plugin-chart-echarts', () => {
+  it('exists', () => {
```
Comment (villebro) [inline superset-frontend/plugins/plugin-chart-echarts/test/Timeseries/index.test.ts:22]: nit: let's replace `describe` and `it` with just `test` blocks.

**Review by villebro — COMMENTED** [review]
Great stuff, this has been a pain for a long time! A few minor comments.

**Review by villebro — APPROVED** [review]
LGTM!

**Comment by michael-s-molina** [comment]
@justinpark I know that you fixed similar problems recently. Could you review this PR?

**Review by kgabryje — APPROVED** [review]

**Pull request merged**